src/Console/Helper/PhingPropertiesHelper.php

```diff
@@ -188,16 +193,26 @@ private function getAllSettings($buildfile = '')
             $buildfile = $this->findPhingBuildFile();
         }
         if ($buildfile) {
+            \Phing::startup();
```

Comment:
Looks fine to me.

Reply:
It's ugly because phing uses includes instead of PSR4 :(